clients/fides-js/src/services/fides/consent.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  PrivacyExperience,
+  PrivacyPreferencesCreateWithCode,
+} from "~/lib/consent-types";
+import { debugLog } from "../../lib/consent-utils";
+
+/**
+ * Fetch the relevant experience based on user location and user device id (if exists).
+ * Fetches both Privacy Center and Overlay components, because GPC needs to work regardless of component
+ */
+export const getExperience = (
+  userLocationString: String,
+  debug: boolean
+): PrivacyExperience | undefined => {
+  debugLog(debug, "Fetching experience for location...", userLocationString);
+  // TODO: GET /privacy-experience
+  return undefined;
+};
+
+/**
+ * Sends user consent preference downstream to Fides
+ */
+export const saveUserPreference = (
```

Comment:
yes I'll update once https://github.com/ethyca/fides/pull/3291 is merged in!

Reply:
oh so we should actually call `saveUserPreference` from the new func here- https://github.com/ethyca/fides/blob/168ed0c74d2dda529a32ae2aba8be87d6a78d296/clients/fides-js/src/lib/preferences.ts#L35

Reply:
updated!

Reply:
ah, got it! I think it might help to call this func something like `patchUserPreferenceToFidesServer` just to be really explicit about it (since there are so many preferences being saved, such as to cookies and to windows). Also, maybe this file should be called `api.ts` since it seems like it's meant to interface with the fides server?